doc/build-macos.md

```diff
@@ -18,18 +18,18 @@ Then install [Homebrew](https://brew.sh).
 Dependencies
 ------------
 
-    brew install automake berkeley-db4 libtool boost --c++11 miniupnpc openssl pkg-config qt
+        brew install automake berkeley-db4 libtool boost --c++11 miniupnpc openssl pkg-config qt
 
-If you want to build the disk image with `make deploy` (.dmg / optional), you need RSVG
+To build .app and .dmg files with, make deploy, you will need RSVG installed.
 
-    brew install librsvg
+        brew install librsvg
 
 NOTE: Building with Qt4 is no longer supported. Build with Qt5.
 
 Build Gridcoin
 --------------
 
-1. Clone the Gridcoin source code and cd into `Gridcoin-Research`
+1. Clone the Gridcoin source code and cd into /Gridcoin-Research.
```

Comment:
`Gridcoin-Research` is fine, `/Gridcoin-Research` will try to `cd` into a root directory.